### ISSUE TITLE
my.cnf symlink turns into None?

### ISSUE THREAD

**Issue by TheLQ** [dev-sec/ansible-mysql-hardening#24]
I think this is related to the symlink reported in #20, on Debian I get to this task and it fails

```
TASK [ansible-mysql-hardening : check include-dir directive is present in my.cnf] ***
fatal: [www-front]: FAILED! => {"changed": false, "failed": true, "msg": "Destination /etc/mysql/my.cnf does not exist !", "rc": 257}
```

The reason is somehow "protect my.cnf" sets the my.cnf symlink to None. If I put pause tasks before and after that task, before it looks like this

```
lrwxrwxrwx 1 root  root   11 Dec  2 18:17 my.cnf -> mariadb.cnf
```

After the task runs 

```
lrwxrwxrwx 1 root  root    4 Dec  2 18:18 my.cnf -> None
```

This is reproducible on the target host on a fresh mysql installation. I'm not sure what's causing it. I fixed it by removing the symlink and copying mariadb.cnf to my.cnf

My config

```
- hosts: www-front
  vars:
    mysql_root_password: "myPassword"
  roles:
    - ansible-mysql-hardening
```

Information

 - Debian Stretch up-to-date
 - Using latest commit: d73fa4ffe93890dcd63f58d68a6b7e9d38f8bd05
 - Ansible 2.2.0.0

**Comment by rndmh3ro** [comment]
Hi @TheLQ, we did not test the role on Debian 9 yet, but I'll try it now.

**Comment by rndmh3ro** [comment]
So I finally took a look.
Problem is that on Debian 9 the my.cnf is a symlink whose target is another symlink. Ansible can't handle this as it seems. Addind state=file should fix this.

**Comment by rndmh3ro** [comment]
Should be fixed with #26

**Comment by rndmh3ro** [comment]
@TheLQ should be fixed now.
